docs/src/main/tut/service.md

```diff
@@ -22,6 +22,8 @@ libraryDependencies ++= Seq(
   "org.http4s" %% "http4s-blaze-server" % http4sVersion,
   "org.http4s" %% "http4s-blaze-client" % http4sVersion
 )
+
+scalacOptions ++= Seq("-Ypartial-unification")
```

Comment:
There's a [plugin](https://github.com/fiadliel/sbt-partial-unification) that covers a few more cases.  Namely, it uses the plugin for versions of Scala that predate the flag.  Would we be better using that?

Reply:
The plugin also works on 2.10.6, though that only helps a subset of the subset on Scalaz.  Yeah, I'll go along with this.

I might put in a plug for sbt-tpolecat, which does this _and_ more.

Reply:
I recommend another PR. That seems outside of the scope of this PR and could get some debate.